pkg/proxy/ipvs/proxier.go

```diff
@@ -1825,27 +1814,36 @@ func (proxier *Proxier) createAndLinkKubeChain() {
 }
 
 // After a UDP or SCTP endpoint has been removed, we must flush any pending conntrack entries to it, or else we
-// risk sending more traffic to it, all of which will be lost (because UDP).
+// risk sending more traffic to it, all of which will be lost.
 // This assumes the proxier mutex is held
+// TODO: move it to util
```

Comment:
this is the first candidate to consolidate, only has the exec dependency, but better in a follow up